commands/project/deploy.ts

```diff
@@ -135,24 +151,21 @@ exports.handler = async options => {
 
     const { data: deployResp } = await deployProject(
       derivedAccountId,
-      projectName,
+      projectName!,
       buildIdToDeploy,
       useV3Api(projectConfig?.platformVersion)
     );
 
-    if (!deployResp || deployResp.error) {
-      logger.error(
-        i18n(`${i18nKey}.errors.deploy`, {
-          details: deployResp.error.message,
-        })
-      );
+    // TODO: Validate this change. Were we actually using deployResp.error.message?
```

Comment:
I have a question about this one. `deployResp.error` doesn't exist on the response object, but we were logging out `deployResp.error.message`. If `deployResp` doesn't exist, then we'd be getting JS errors. So I assume this was never being triggered, but I wanted to double check just in case.

Reply:
Seems safe to get rid of!